cmd/root.go

```diff
@@ -80,4 +86,14 @@ func init() {
 		"frequency-plans-url",
 		"https://raw.githubusercontent.com/TheThingsNetwork/lorawan-frequency-plans/master",
 		"URL for fetching frequency plans")
+
+	rootCmd.AddCommand(ttnv2.TTNv2Cmd)
+	rootCmd.AddCommand(ttnv3.TTNv3Cmd)
+	rootCmd.AddCommand(chirpstack.ChirpStackCmd)
+
+	// // Add when `github.com/spf13/cobra` package is updated
+	// rootCmd.AddGroup(&cobra.Group{
+	// 	ID:    "sources",
+	// 	Title: "Sources",
+	// })
```

Comment:
File issue and reference with TODO

Reply:
Filed an issue https://github.com/TheThingsNetwork/lorawan-stack-migrate/issues/72 and added `TODO:` in code comments